slam-libraries/viam-cartographer/src/mapping/map_builder.cc

```diff
@@ -57,6 +57,30 @@ void MapBuilder::BuildMapBuilder() {
         cartographer::mapping::CreateMapBuilder(map_builder_options_);
 }
 
+void MapBuilder::LoadFromFile(std::string map_filename,
+                              bool load_frozen_trajectory, bool optimize) {
+    std::map<int, int> trajectory_ids_map =
+        map_builder_->LoadStateFromFile(map_filename, load_frozen_trajectory);
+
+    if (optimize) {
+        map_builder_->pose_graph()->RunFinalOptimization();
+    }
+    for (auto&& trajectory_ids_pair : trajectory_ids_map)
+        LOG(INFO) << "Trajectory ids mapping: " << trajectory_ids_pair.first
```

Comment:
We're not using BOOST because cartographer uses glog and this way we can have one standardized logging framework to forward all logs for viam-cartographer.

Reply:
Cool! Thanks for the clairification!